k8s/namespaces/sscs/sscs-evidence-share/demo.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
 spec:
   values:
     java:
-      image: hmctspublic.azurecr.io/sscs/evidence-share:prod-4b38aa2-20220519123949 #{"$imagepolicy": "flux-system:sscs-evidence-share"}
```

Comment:
Might be worth keeping the image line here for demo as it changes pretty frequently